server/src/server.rs

```diff
@@ -171,13 +171,17 @@ impl CubebServer {
         CubebServer {
             cb_remote: cb_remote,
             streams: StreamSlab::with_capacity(STREAM_CONN_CHUNK_SIZE),
+            remote_pid: None,
```

Comment:
`remote_pid` will be updated every time when `ClientContext` is created. Would `last_remote_pid` be a better name ?